new/LIT6848ZemmareDabraZa.xml

```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6848ZemmareDabraZa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዝማሬ፡ ዘደብረ፡ ዘይት፡</title>
+                <title corresp="#t1" xml:lang="gez" type="normalized">Zəmmāre za-Dabra Zayt</title>
+                <title corresp="#t1" xml:lang="en">Chant to the Mount of Olives</title>
```

Comment:
I would translate it "Chant for Dabra Zayt". Because the idea is "Chant for (the feast of) Dabra Zayt"

Reply:
Yes, I agree.